src/azure-cli/azure/cli/command_modules/synapse/_help.py

```diff
@@ -575,14 +575,14 @@
         --resource-group rg --key-identifier https://{keyvaultname}.vault.azure.net/keys/{keyname}
 """
 
-helps['synapse workspace key update'] = """
+helps['synapse workspace key activate'] = """
 type: command
-short-summary: Update a workspace's key.
-long-summary: Activate a workspace and change it's state from pending to success state when the workspace is first being provisioned through updating a workspace's key. Switching to another CMK after activating the workspace needs to execute 'az synapse workspace update' instead.
+short-summary: active a workspace's key.
+long-summary: Activate a workspace and change it's state from pending to success state when the workspace is first being provisioned through activating a workspace's key.
```

Comment:
I think we can improve the summary by referencing the following docs:
1. https://github.com/Azure/azure-powershell/blob/main/src/Synapse/Synapse/help/Enable-AzSynapseWorkspace.md
2. https://docs.microsoft.com/en-us/azure/synapse-analytics/security/workspaces-encryption

Basically, this command is used to activate a workspace rather than a workspace key